app/views/movies/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="container pt-5">
+  <div class="movie-page d-flex mt-5 py-5">
+    <img src="<%= @movie.poster_url %>" alt="Movie image" class="movie-image d-flex">
```

Comment:
Need to make this responsive

Reply:
Could make responsive changes in a new branch

Reply:
Yes, okay, will do! Thanks!